.github/workflows/tests.yml

```diff
@@ -71,8 +67,10 @@ jobs:
             php-version: '7.4'
           - symfony: '6.4.*'
             php-version: '8.2'
-          - symfony: '7.0.*'
-            php-version: '8.2'
+          - symfony: '7.4.*'
+            php-version: '8.3'
```

Comment:
the build error might be related to what happens when we use php 8.3 instead of 8.2, or because we require 7.4 instead of 7.0.  7.4 has no stable release yet, do we need to allow dev versions for the 7.4 and 8.0 build?

Reply:
I'm not sure why 7.4 fail ; I'll try to remove 7.4 first to see if it works on 8.0 first.

Anyway I discover it will be block by https://github.com/phpspec/phpspec